sql/moz-fx-data-shared-prod/bigeye_derived/dashboard_service_v1/query.py

```diff
@@ -0,0 +1,169 @@
+"""Pull from BigEye API from dashboardService and upload to Big Query."""
+
+import json
+import os
+import time
+from typing import Any, Dict
+
+# Libraries specific for pulling from API and uploading to bigquery.
+import pandas as pd
+import requests
+from google.api_core.exceptions import GoogleAPIError
+from google.cloud import bigquery
+from requests.exceptions import RequestException
+
+# Configuration
+TARGET_PROJECT = os.getenv("TARGET_PROJECT", "moz-fx-data-shared-prod")
```

Comment:
I'd suggest we set these up as script arguments to stay consistent with how we do it in other scripts. For example: https://github.com/mozilla/private-bigquery-etl/blob/main/sql/moz-fx-data-shared-prod/cinder_external/jobs_raw_v1/query.py#L95-L108